app/core/core-ui/src/main/kotlin/com/hedvig/android/core/ui/SelectIndicationCircle.kt

```diff
@@ -10,28 +17,54 @@ import androidx.compose.material3.LocalContentColor
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Surface
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.compose.ui.unit.dp
 import com.hedvig.android.core.designsystem.preview.HedvigPreview
 import com.hedvig.android.core.designsystem.theme.HedvigTheme
 import com.hedvig.android.core.ui.preview.BooleanCollectionPreviewParameterProvider
 
 @Composable
-fun SelectIndicationCircle(selected: Boolean, modifier: Modifier = Modifier) {
-  Spacer(
-    modifier
-      .size(24.dp)
-      .clip(CircleShape)
-      .then(
-        if (selected) {
-          Modifier.background(LocalContentColor.current)
-        } else {
-          Modifier.border(2.dp, MaterialTheme.colorScheme.onSurfaceVariant, CircleShape)
-        },
-      ),
-  )
+fun SelectIndicationCircle(
+  selected: Boolean,
+  modifier: Modifier = Modifier,
+  selectedIndicationColor: Color = LocalContentColor.current,
+) {
+  val selectedTransition = updateTransition(selected)
+  selectedTransition.AnimatedContent(
+    transitionSpec = {
+      fadeIn(tween(durationMillis = 500)) togetherWith fadeOut(tween(90))
```

Comment:
I can't find any specifications, but I asked Julia how they do it:)

Reply:
If EaseInOut is used, we might actually want to use something like `MotionTokens.EasingStandardCubicBezier` instead of a `tween` which does it completely linerarly. We don't have an EaseInOut specifically ourselves but maybe we can add one?

Something like this https://cubic-bezier.com/#.6,0,.4,1 perhaps? Maybe William should have an opinion about these things too. We could get some motion tokens too to re-use across the app.

Btw read more here https://medium.com/androiddevelopers/easing-in-to-easing-curves-in-jetpack-compose-d72893eeeb4d about all this if you are not very familiar